sympy/functions/elementary/trigonometric.py

```diff
@@ -2523,9 +2522,6 @@ def _eval_as_leading_term(self, x):
         else:
             return self.func(arg)
 
-    def _eval_is_real(self):
```

Comment:
Why is this one being removed?

Reply:
Duplicated method:

https://github.com/sympy/sympy/blob/2874be74a56332067395b5b2322c3c35faface1d/sympy/functions/elementary/trigonometric.py#L2437-L2438